src/io.rs

```diff
@@ -25,6 +27,8 @@ pub const ALL: Token = Token(0);
 pub const SYSTEM: Token = Token(1);
 const CONN_START: Token = Token(2);
 
+pub type Slab<T> = slab::Slab<T, Token>;
```

Comment:
If you make the following changes it will compile:
``` diff
diff --git a/src/io.rs b/src/io.rs
index 627a4c5..2ea97e5 100644
--- a/src/io.rs
+++ b/src/io.rs
@@ -10,7 +10,6 @@ use mio::{
     PollOpt,
 };
 use mio::tcp::{TcpListener, TcpStream};
-use slab;
 
 use url::Url;
 
@@ -21,14 +20,13 @@ use communication::{Sender, Signal, Command};
 use result::{Result, Error, Kind};
 use connection::Connection;
 use factory::Factory;
+use util::Slab;
 use super::Settings;
 
 pub const ALL: Token = Token(0);
 pub const SYSTEM: Token = Token(1);
 const CONN_START: Token = Token(2);
 
-pub type Slab<T> = slab::Slab<T, Token>;
-
 pub type Loop<F> = EventLoop<Handler<F>>;
 type Conn<F> = Connection<<F as Factory>::Handler>;
 type Chan = mio::Sender<Command>;
diff --git a/src/util.rs b/src/util.rs
index a8dca9d..664ba6d 100644
--- a/src/util.rs
+++ b/src/util.rs
@@ -1,6 +1,9 @@
 //! The util module rexports some tools from mio in order to facilitate handling timeouts.
+use slab;
 
 /// Used to identify some timed-out event.
 pub use mio::Token;
 /// A handle to a specific timeout.
 pub use mio::Timeout;
+/// A Slab allocator for associating tokens to data.
+pub type Slab<T> = slab::Slab<T, Token>;
```

I don't want to remove Slab from util because it's part of the public api.